python/pyarrow/__init__.py

```diff
@@ -44,6 +44,13 @@ def parse_version(root):
         __version__ = None
 
 
+import pyarrow.compat as compat
+
+
+# Workaround for https://issues.apache.org/jira/browse/ARROW-2657
+compat.import_tensorflow_extension()
```

Comment:
Shouldn't this be run only on Linux?

Reply:
Yes, sorry I missed that (it was in the prior iteration of the patch but I didn't look closely enough at this function): https://issues.apache.org/jira/browse/ARROW-2795